### ISSUE TITLE
Windows problem: backslashes returned for css output

### ISSUE THREAD

**Issue by aproc** [borodean/postcss-assets#19]
I'm on a Windows machine and using postcss-assets. When I use resolve() in combination with the 'relativeTo' option, I receive double backslashes as css output instead of a simple /

```scss
/* source input: */
.selector { background-image: resolve('filename.png'); }
```

```css
/* generated output: */
.selector { background-image: url('..\\images\\filename.png'); }
```

```css
/* expected output: */
.selector { background-image: url('../images/filename.png'); }
```

**Comment by borodean** [comment]
Hi @aproc! Thanks for your feedback. I think I can fix it.

The only question is how to make it testable—Travis CI has no option for running tests on Windows.

**Comment by aproc** [comment]
Sorry, I'm not familar with Travis CI. I had a look at your code and tried a quick and dirty way to get the expected result after inspecting the output of  ``` cssesc(url.format(assetUrl))```. But I'm not sure if it has any unwanted side effects. Maybe it helps?

```javascript
/* index.js */
Assets.prototype.resolveUrl = function (assetStr) {
...
  /* line: 130 */
  return cssesc(url.format(assetUrl));

  /* change works on windows */
  return cssesc(url.format(assetUrl.path));
}



```

**Comment by borodean** [comment]
Fixed in [2.1.1](https://github.com/borodean/postcss-assets/releases/tag/2.1.1).
